Makefile

```diff
@@ -28,7 +28,7 @@ $(OUT_DIR):
 .PHONY: build
 build: $(OUT_BIN)
 
-$(OUT_BIN): $(BPF_OBJ) $(LIBBPF_OBJ) $(GO_SRC) | $(OUT_DIR)
+$(OUT_BIN): $(LIBBPF_OBJ) $(GO_SRC) | $(OUT_DIR)
```

Comment:
Note that default build is OUT_BIN - meaning that if you remove this dependency, you should add the BPF_OBJ target to PHONY build (or build all that you removed)